src/main/java/de/uni_tuebingen/ub/nppm/db/DeleteDB.java

```diff
@@ -0,0 +1,35 @@
+package de.uni_tuebingen.ub.nppm.db;
+
+import static de.uni_tuebingen.ub.nppm.db.AbstractBase.getSession;
+import org.hibernate.Session;
+import org.hibernate.query.NativeQuery;
+
+public class DeleteDB extends AbstractBase {
```

Comment:
Introducing a new class named `DeleteDB` and adding methods in it that do select & update-Operations doesn't really make sense. SELECT & UPDATE has nothing to do with delete. 

Maybe we just move these functions to e.g. DatenbankDB?